src/parser/mod.rs

```diff
@@ -6083,6 +6083,24 @@ impl<'a> Parser<'a> {
                 None
             };
             Ok(Some(ColumnOption::Identity(property)))
+        } else if dialect_of!(self is SQLiteDialect)
+            && self.parse_keywords(&[Keyword::ON, Keyword::CONFLICT])
+        {
+            // Support ON CONFLICT for SQLite
+            let on_confl = self.parse_one_of_keywords(&[
+                Keyword::ROLLBACK,
+                Keyword::ABORT,
+                Keyword::FAIL,
+                Keyword::IGNORE,
+                Keyword::REPLACE,
+            ]);
+            match on_confl {
+                Some(keyword) => Ok(Some(ColumnOption::OnConflict(keyword))),
+                _ => self.expected(
+                    "one of ROLLBACK, ABORT, FAIL, IGNORE or REPLACE",
+                    self.peek_token(),
+                ),
```

Comment:
It should be good to use unreadable here since `parse_one_of_keywords` has guarded the keyword.
```suggestion
                _ => unreachable!()
```

Reply:
I saw that unreachable is often used through the rest of the parser, I modified the code accordingly.